ui/page/wallets/privacy_page.go

```diff
@@ -446,5 +447,6 @@ func (pg *PrivacyPage) listenForMixerNotifications() {
 // components unless they'll be recreated in the OnNavigatedTo() method.
 // Part of the load.Page interface.
 func (pg *PrivacyPage) OnNavigatedFrom() {
-	pg.ctxCancel()
+	pg.WL.MultiWallet.RemoveAccountMixerNotificationListener(PrivacyPageID)
+	close(pg.MixerChan)
```

Comment:
I might have missed this while reviewing previous pages with similar setup, but please apply this wherever a page has a ctx that is created `OnNavigatedTo` and canceled `OnNavigatedFrom`. The listener should be removed after the context is canceled, by the same goroutine that checks if the ctx is canceled via `case <-ctx.Done():`. Only then should the listener be removed and the channel closed. The ctx should always be canceled in `OnNavigatedFrom`, to ensure the goroutine that was started to listen for the notifications is not left running indefinitely.